tests/phpunit/tests/KSES.php

```diff
@@ -326,6 +369,10 @@ public function data_test_filter_kses_allowed_html() {
 				'<div class="foo" id="bar" style="color: pink" role="main" title="Test"></div>',
 				'<div class="foo" id="bar" style="color: pink" role="main" title="Test"></div>',
 			],
+			'Img Attributes'       => [
+				'<img src="http://www.example.com/test.jpg" alt="Example" height="200" width="500" />',
+				'<img src="http://www.example.com/test.jpg" alt="Example" height="200" width="500" />',
+			],
```

Comment:
And just to be safe, this test fails without the change in this PR?

Reply:
Both adding 'width' and `array_merge_recursive_distinct` fix the issue alone. I think we should add `array_merge_recursive_distinct` still, as it will fix other issues in the future.

Reply:
Sure, yeah we can keep both.

Just wanted to make sure that this test works as expected, i.e. it should fail on `main`.

And I just confirmed that locally.

So all good 👍